CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.5.0] - 2018-05-15
+### Security
+- update direct dependencies flagged as `Critical` and `High` vulnerabilities by npm audit
+- this update causes the following breaking change: publish-please must run on node version >= 6.0.0
```

Comment:
Since we follow semantic versioning I guess we should bump major version due to the breaking change.